<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleBar</class>
 <widget class="QWidget" name="TitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="rightMargin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,10,0,1">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="labelLogo">
       <property name="minimumSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnMinisized">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMaxsized">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
